src/layer/poi.js

```diff
@@ -0,0 +1,81 @@
+import * as label from "../constants/label.js";
+import * as Color from "../constants/color.js";
+
+const poiIcon = [
+  "match",
+  ["get", "class"],
+  ["bar", "beer"],
+  "poi_martini_glass",
+  "hospital",
+  "poi_health_cross",
```

Comment:
Did you want to use an “H” instead of a ➕ for hospitals, per https://github.com/ZeLonewolf/openstreetmap-americana/pull/387#discussion_r917476571?

Reply:
Done :+1: